### PR TITLE
Project Summary

### PR DESCRIPTION
**Pull request by jeremiehelme**
Adapted the project summary popover on the map to the mockup

project_summary_before

project_summary_after

### CODE REVIEW

**Comment by SamMousa** [comment]
Please add a description to the PR so that I know what it is supposed to do.
Screenshots before / after are appreciated as well.

**Comment by SamMousa** [comment]
There are a lot of HTML structural changes that should not be there.
Wrapping a div in a div just for markup is usually not the correct solution.

**Comment by jeremiehelme** [comment]
Added the workspace button to the popover

Capture d’écran 2020-02-18 à 15 58 50

File: protected/views/project/summary.php [hunk 1]
```diff
@@ -338,7 +386,10 @@
     }
 
     if (!empty($project->pages) && (true || \Yii::$app->user->can(Permission::PERMISSION_READ, $project))) {
-       echo Html::a('Details', ['project/view', 'id' => $project->id], ['target' => '_top']);
+        echo Html::beginTag('div', ['class' => 'actions']);
```
Comment (SamMousa) [inline protected/views/project/summary.php]: The buttons are in the grid, there should be no need for extra tags just to create boxes?

**Comment by luna404** [comment]
@jeremiehelme looks good, but we might have to find better icons or labels for the two buttons to make it more intuitive for users.
@SamuelPetragallo what do you think of renaming the two buttons to `Data` and `Dashboard`?

**Comment by SamMousa** [comment]
Doesn't the button take them to the workspaces page?
Why not call it Workspaces & Dashboard?

Side note @jeremiehelme : we have an icon for the dashboard, it is used in the project list; so you could have 2 icons both accompanied with text.

**Comment by jeremiehelme** [comment]
@luna404 @SamMousa I agree, having a label and an icon for both is better. That way you can identify the action with one or the other.

For the label, I agree with Sam, calling it Workspaces and Dashboard make the most sense to me 

Capture d’écran 2020-02-25 à 14 42 19

**Comment by SamuelPetragallo** [comment]
Agree on using "Workspaces" and "Dashboard", would simply invert both as majority of users will go to Dashboards

**Comment by jeremiehelme** [comment]
If the majority of users use the Dashboard button then it makes sense to me that it is larger than the other ?
Ok to invert them

**Comment by luna404** [comment]
@jeremiehelme is this intended to be implemented as part of this week's release? If so, we need to have an opportunity to test it on staging today :)

**Comment by SamMousa** [comment]
It still has some unresolved issues so have removed it from the milestone

**Comment by jeremiehelme** [comment]
@SamMousa @luna404 Sorry, I didn't see it was due this week, here it is

**Pull request closed without merging**